SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Events/ShouldSendFeedbackEventHandler.cs

```diff
@@ -2,5 +2,5 @@
 
```

Comment:
Should we increase the last digit from a manifest file? I think we already send a version for testing.

Reply:
done